src/modules/dashboard/components/ColonyEvents/ColonyEvents.tsx

```diff
@@ -33,6 +35,15 @@ const MSG = defineMessages({
     id: 'dashboard.ColonyEvents.labelFilter',
     defaultMessage: 'Filter',
   },
+
+  transactionsLogTitle: {
+    id: 'dashboard.ColonyActions.transactionsLogTitle',
```

Comment:
This id should be prefaced with `dashboard.ColonyEvents.` instead of `dashboard.ColonyActions` right? Same with the id of the message below this one